unified_targets/docs/Manufacturers.md

```diff
@@ -24,6 +24,7 @@ This is the official list of manufacturer ids (`manufacturer_id` in the target c
 |JHEF|JHE\_FPV|https://github.com/atgfpeyv|
 |KLEE|Kevin Lee (WhitenoiseFPV)|https://whitenoisefpv.com/|
 |MERA|MerakRC|http://www.merakrc.com/|
+|MOLA|MotoLab|https://github.com/MotoLab|
```

Comment:
Seems like the manufacturer ID should be `MOTO` but maybe this was already agreed on to be `MOLA`?

Reply:
I chose 'MOLA' based on the facts that the name is camel cased, and the existing targets are called ML... Do you think 'MOTO' is more accurate? (Also, I do not think that this is extremely consequential, at some point we will probably want to have a list of manufacturer ids, with full name and website link, built into configurator.)

Reply:
I don't think it's a big deal either way. Just that "MOTO" would make me think of Motolab and my initial reaction was confusion about what was "MOLA". Also with the target names being shortened to "ML" I didn't initially get the Motolab connection. Took me a bit to connect the dots with "MLTEMP" being Motolab Tempest.

Reply:
Hehe, at least they are consistently confusing now. :stuck_out_tongue_closed_eyes: